infra/data-pipeline.yml

```diff
@@ -1333,6 +1333,7 @@ tasks:
       dependencies:
         - docker-build
         - integration-test
+        - test-label-scheduling-history-generator
```

Comment:
When we will also download the model, we can remove this and just make the build task depend on the trainer task